lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -1273,4 +1282,12 @@ private function getOldColumnComment(ColumnDiff $columnDiff) : ?string
     {
         return $columnDiff->fromColumn ? $this->getColumnComment($columnDiff->fromColumn) : null;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function escapeStringForObject($data)
+    {
+        return pg_escape_bytea($data);
```

Comment:
As far as I can tell, this function should be used to represent the data as a string literal in SQL. It's not needed when used as a parameter for a prepared statement.